bolt/lib/Rewrite/BinaryPassManager.cpp

```diff
@@ -398,7 +396,7 @@ Error BinaryFunctionPassManager::runAllPasses(BinaryContext &BC) {
                          opts::StripRepRet);
 
   Manager.registerPass(std::make_unique<IdenticalCodeFolding>(PrintICF),
-                       opts::ICF);
+                       opts::ICF != IdenticalCodeFolding::ICFLevel::None);
```

Comment:
nit: define a helper method `bool operator()` to avoid spelling this check in several places?